tests/conftest.py

```diff
@@ -158,6 +158,26 @@ def mock_get_request(path: str, unifi_path: str, payload: dict[str, Any]) -> Non
     )
 
 
+@pytest.fixture(name="_mock_traffic_rule_endpoint")
+def endpoint_fixture(
+    mock_aioresponse: aioresponses,
+    is_unifi_os: bool,
+    traffic_rule_payload: dict[str, Any],
+) -> None:
+    """Use fixtures to mock traffic rule endpoint."""
+
+    def mock_get_request(path: str, unifi_path: str, payload: dict[str, Any]) -> None:
+        """Register HTTP response mock."""
+        url = unifi_path if is_unifi_os else path
+        data = payload
+        mock_aioresponse.get(f"https://host:8443{url}", payload=data)
+
+    mock_get_request(
+        "/proxy/network/v2/api/site/default/trafficrules",
+        "/proxy/network/v2/api/site/default/trafficrules",
```

Comment:
One should be for devices running unifi os and one should be for self hosted, Im guessing its just removing /proxy/network
```suggestion
        "/v2/api/site/default/trafficrules",
        "/proxy/network/v2/api/site/default/trafficrules",
```

Reply:
Already made this adjustments locally. Thanks for the feedback.